### PR TITLE
add missing system function

### PR DESCRIPTION
**Pull request by jasontho-ms**
# Pull Request Template

## Description

Add a missing system function

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Closing issues

Put closes #XXXX in your comment to auto-close the issue that your PR fixes (if such).

## Assignee

Please add yourself as the assignee

## Porjects

Please add relevant projects so this issue can be properly tracked.

### CODE REVIEW

File: Microsoft.Azure.Cosmos/src/SqlObjects/SqlFunctionCallScalarExpression.cs [hunk 1]
```diff
@@ -15,6 +15,7 @@ internal sealed class SqlFunctionCallScalarExpression : SqlScalarExpression
         {
             { Names.InternalCompareBsonBinaryData, Identifiers.InternalCompareBsonBinaryData },
             { Names.InternalCompareObjects, Identifiers.InternalCompareObjects },
+            { Names.InternalObjectToArray, Identifiers.InternalObjectToArray },
```
Comment (bchong95) [inline Microsoft.Azure.Cosmos/src/SqlObjects/SqlFunctionCallScalarExpression.cs]: Please add these to the `SqlObjectVisitorBaselineTests`.

Comment (jasontho-ms) [reply Microsoft.Azure.Cosmos/src/SqlObjects/SqlFunctionCallScalarExpression.cs]: Alas, that test infrastructure is not part of the V3 SDK

Comment (j82w) [reply Microsoft.Azure.Cosmos/src/SqlObjects/SqlFunctionCallScalarExpression.cs]: @bchong95 is there anything stopping us from porting the tests?

**Comment by kirankumarkolli** [comment]
Rebase screwed it-up

**Comment by kirankumarkolli** [comment]
Dupe of #522

**Pull request closed without merging**